src/python/pants/backend/python/packaging/pyoxidizer/rules.py

```diff
@@ -185,8 +186,17 @@ async def package_pyoxidizer_binary(
     )
 
 
+@rule
```

Comment:
Literal copy/paste from Go and only re-named the field_set.

Begs the question of whether these run commands can be made implicit and auto-generated with a custom `is_executable` field or some sort of hook, without even needing these 6 lines.

Reply:
Sweet. Not yet at least, for `pex_binary` we go through a ton of trouble to make building the PEX faster. For example, if you change a source file we don't rebuild the PEX. This doesn't generalize to all implementations.